runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.utils.ts

```diff
@@ -23,12 +24,14 @@ interface BaseCredentialChangeMailPayload {
     collectionCreatorEmail: string;
     collectionName: string;
     credentialName: string;
+    credentialOldName?: string | undefined;
```

Comment:
If undefined is allowed, and this is the only use of the interface, why to make credentialOldName optional?

Reply:
You're right